src/components/DataTableData/DataTableData.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const DataTableData = props => {
+  const { children, className, checked, value, id, expanded, ...other } = props;
+
+  let content;
+  if (!(checked === undefined) && !(expanded === undefined)) {
+    content = '';
+  } else if (checked !== undefined) {
+    content = (
+      <span>
+        <input
+          checked={checked}
+          id={id}
+          className="bx--checkbox"
+          type="checkbox"
+          value={value}
+          name={id}
+        />
+        <label htmlFor={id} className="bx--checkbox-label">
+          <span className="bx--checkbox-appearance">
+            <svg
```

Comment:
I can switch the expandable icon, but for the checkmark one I needed to add the full HTML because our `Checkbox` component gets wrapped in a `bx--form-item` element.